lib/api.db.mysql.js

```diff
@@ -28,7 +28,7 @@ if (api.mysql) {
   api.db.mysql.open = function(database, callback) {
     database.retryCounter++;
     var application = database.application,
-        connection = api.mysql.createConnection(database.url);
```

Comment:
When msql configured with closed TCP/IP. I needed to use connection via socet, so I can't use string url in this case.
socketPath: The path to a unix domain socket to connect to. When used host and port are ignored.